mix.exs

```diff
@@ -44,7 +44,7 @@ defmodule Guardian.Mixfile do
   defp elixirc_paths(_), do: ["lib"]
 
   def application do
-    [applications: [:logger, :poison, :jose, :elixir_uuid]]
+    [applications: [:crypto, :logger, :poison, :jose]]
```

Comment:
We dont need this anymore.

Use `extra_applications: [:logger]`

Reply:
Yep, good point.